feature-detects/svg/externalresources.js

```diff
@@ -0,0 +1,18 @@
+/*!
+{
+  "name": "SVG External Resources",
+  "property": "externalresources",
+  "authors": ["Moritz Schramm"],
+  "tags": ["svg"]
+}
+!*/
+/* DOC
+Detects support for External Resources in SVG.
+
+See [this discussion](https://github.com/Modernizr/Modernizr/issues/1409) regarding SVG external content.
+*/
+define(['Modernizr'], function(Modernizr) {
+  Modernizr.addTest('externalresources', function() {
+    return document.implementation.hasFeature('http://www.w3.org/TR/SVG11/feature#ExternalResourcesRequired', '1.1');
```

Comment:
Doesnt that violate  https://github.com/Modernizr/Modernizr/issues/1409#issuecomment-51089302 : 
"unfortunately, modernizr can't really use any external uris, ever. all of our tests need to be self contained and not realiant upon a network connection."